quimb/gen/operators.py

```diff
@@ -343,17 +343,63 @@ def fsim(theta, phi, dtype=complex, **kwargs):
     a = cos(theta)
     b = -1j * sin(theta)
     c = exp(-1j * phi)
-
     gate = [[1, 0, 0, 0],
             [0, a, b, 0],
             [0, b, a, 0],
             [0, 0, 0, c]]
 
+
+
     gate = qu(gate, dtype=dtype, **kwargs)
```

Comment:
1. Just a general note - it is best to try and only commit changes to functions you are adding/modifying
2. Also the code in `quimb` is [pep8 compliant](https://www.python.org/dev/peps/pep-0008/) meaning there are general rules about how much space should be used between lines. If you run a tool like [`pylama`](https://github.com/klen/pylama) it will guide you (or most editors these days too).